test/rest_api/test_compaction_manager.py

```diff
@@ -15,6 +15,50 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
 
+import sys
+
+from requests import HTTPError
+
+# Use the util.py library from ../cql-pytest:
+sys.path.insert(1, sys.path[0] + '/../cql-pytest')
+from util import new_test_table
+
 def test_compaction_manager_stop_compaction(rest_api):
     resp = rest_api.send("POST", "compaction_manager/stop_compaction", { "type": "COMPACTION" })
+
+def test_compaction_manager_stop_keyspace_compaction(cql, rest_api, test_keyspace):
+    resp = rest_api.send("POST", f"compaction_manager/stop_keyspace_compaction/{test_keyspace}", { "type": "RESHAPE" })
     resp.raise_for_status()
+
+    # non-existing keyspace
+    resp = rest_api.send("POST", f"compaction_manager/stop_keyspace_compaction/{test_keyspace}XXX", { "type": "RESHAPE" })
+    try:
+        resp.raise_for_status()
+    except HTTPError:
+        pass
+
+def test_compaction_manager_stop_keyspace_compaction_tables(cql, rest_api, test_keyspace):
+    with new_test_table(cql, test_keyspace, "a int, PRIMARY KEY (a)") as t0:
+        test_tables = [t0 if not '.' in t0 else t0.split('.')[1]]
+        resp = rest_api.send("POST", f"compaction_manager/stop_keyspace_compaction/{test_keyspace}", { "tables": f"{test_tables[0]}", "type": "CLEANUP" })
+        resp.raise_for_status()
+
+        # non-existing table
+        resp = rest_api.send("POST", f"compaction_manager/stop_keyspace_compaction/{test_keyspace}", { "tables": "foo", "type": "CLEANUP" })
+        try:
+            resp.raise_for_status()
+        except HTTPError:
+            pass
+
+        # multiple tables
+        with new_test_table(cql, test_keyspace, "b int, PRIMARY KEY (b)") as t1:
+            test_tables += [t1 if not '.' in t1 else t1.split('.')[1]]
```

Comment:
ditto - unnecessary if, will always be the else case

Reply:
done